server/util/timeseries/timeseries.go

```diff
@@ -0,0 +1,88 @@
+package timeseries
```

Comment:
can you send this separately first?

Reply:
sent https://github.com/buildbuddy-io/buildbuddy/pull/7810/files